src/diffusion_for_multi_scale_molecular_dynamics/models/axl_diffusion_lightning_model.py

```diff
@@ -254,18 +271,20 @@ def _generic_step(
             f"Got shape = {atom_shape}"
         )
 
-        l0 = batch[
-            "box"
-        ]  # should be batch[UNIT_CELL] - see later comment with batch['box']
-        # TODO assert on shape
+        l0 = batch[LATTICE_PARAMETERS]
+        lattice_shape = l0.shape
+        assert len(lattice_shape) == 2, (
```

Comment:
I like that the error message in the `assert` basically acts as a comment in the code to tell the reader what shapes to expect. 

Please remove the comment on line 246 
> # The RELATIVE_COORDINATES have dimensions [batch_size, number_of_atoms, spatial_dimension].
which is no longer needed.

Reply:
fixed